backend/de.metas.banking/de.metas.banking.swingui/src/main/java/de/metas/banking/payment/paymentallocation/model/InvoiceRow.java

```diff
@@ -533,7 +535,7 @@ else if (invoiceRow.getC_Order_ID() > 0)
 		final CurrencyId currencyId = currenciesRepo.getByCurrencyCode(invoiceRow.getCurrencyISOCode()).getId();
 
 		return PayableDocument.builder()
-				.orgId(invoiceRow.getOrgId())
+				.clientAndOrgId(ClientAndOrgId.ofClientAndOrg(ClientId.METASFRESH, invoiceRow.getOrgId()))
```

Comment:
avoid avoid hardcoding ClientId

Reply:
Problem: i don't have a client id here and i believe org is null as well as it is never set to the builder. 
I guess this never failed because it was never used.
Is there a point in investing any time in fixing this, since this belongs to swing?

Can i use `Env.getClientId()` and `Env.getOrgId()` to get some generic values?

Reply:
agree. don't invest time in swing.

Reply:
fixed